src/settings-ui/Microsoft.PowerToys.Settings.UI/Views/FancyZonesPage.xaml

```diff
@@ -98,31 +98,43 @@
                                             HorizontalAlignment="Right"/>
                                     </controls:Setting.ActionContent>
                                 </controls:Setting>
+                            </StackPanel>
+                        </controls:SettingExpander.Content>
+                    </controls:SettingExpander>
+                    
+                    <controls:SettingExpander IsExpanded="True">
+                        <controls:SettingExpander.Header>
+                            <controls:Setting x:Uid="FancyZones_UseCustomColors" Icon="&#xE790;" Style="{StaticResource ExpanderHeaderSettingStyle}">
```

Comment:
I'd make a minor change here:

Maybe use a ComboBox instead of a ToggleSwitch to make things less ambiguous and inline with W11 settings. For e.g.:

Header: Zone background colors
Option 1: Windows accent colors
Option 2: Manual

If manual is selected we show the 3 colors. If not these would be collapsed.

You could use a converter to turn the bool into the selectedindex. There are some examples of that in the variius settings pages.

Reply:
Ok! I will update the PR.
I like this since is also the same approach adopted shortcut guide activation.